src/components/materialization/create/index.tsx

```diff
@@ -26,18 +26,19 @@ import ErrorBoundryWrapper from 'components/shared/ErrorBoundryWrapper';
 import PageContainer from 'components/shared/PageContainer';
 import { useConfirmationModalContext } from 'context/Confirmation';
 import { useClient, useQuery, useSelect } from 'hooks/supabase-swr';
+import { usePrompt } from 'hooks/useBlocker';
 import useBrowserTitle from 'hooks/useBrowserTitle';
 import { DraftSpecQuery } from 'hooks/useDraftSpecs';
 import { useZustandStore } from 'hooks/useZustand';
-import { MouseEvent, useEffect } from 'react';
+import { isEmpty } from 'lodash';
+import { MouseEvent } from 'react';
 import { FormattedMessage, useIntl } from 'react-intl';
 import { useNavigate } from 'react-router-dom';
 import { TABLES } from 'services/supabase';
 import useNotificationStore, {
     Notification,
     NotificationState,
 } from 'stores/NotificationStore';
-import { MaterializationDef } from '../../../../flow_deps/flow';
```

Comment:
Switching this type to `any` just to prototype some stuff. I'll switch it back in a follow up PR